krm-functions/nad-fn/testdata/golden/create_full_resources/interface-n3.yaml

```diff
@@ -8,5 +8,5 @@ spec:
   networkInstance:
     name: vpc-ran
   cniType: sriov
-  attachementType: vlan
+  attachmentType: vlan
```

Comment:
Do the Go type and CRD for these (if there is one) all have the right name?

Reply:
I think it's spelled correctly there.
https://github.com/nephio-project/api/blob/main/nf_requirements/v1alpha1/interface_types.go